config/application.rb

```diff
@@ -28,5 +28,14 @@ class Application < Rails::Application
 
     # Autoloadable lib directories with classes and modules.
     config.autoload_paths += %W(#{config.root}/lib)
+
+    # Spristupnenie konstant
+    config.before_configuration do
+      env_file = File.join(Rails.root, 'config', 'local_env.yml')
+      YAML.load(File.open(env_file)).each do |key, value|
```

Comment:
Tie vypocty uz mam vsetky v lib, ale problem je, ze nie vsetky konstanty suvisia iba s tymito vypoctami. Mohla by som to spravit aj tak, ze konstanty, ktore suvisia s vypoctami budu v lib a dalsie dve v modeli? Lebo napr. NUMBER_LOS_IN_ROOM pouzivam na viacerych miestach.